### ISSUE TITLE
Persist split Terminal

### ISSUE THREAD

**Issue by bounty1342** [microsoft/vscode#95984]
Creating a VS code workspace with 3 projects, I would like to have 3 terminals matching projet location that persist on restart.

Please find an exemple below :

![image](https://user-images.githubusercontent.com/6307253/80125739-5d953380-8592-11ea-8686-19aef3b35a8e.png)

Regards,

**Comment by Tyriar** [comment]
So this is 2 different things depending on what you mean by restart.

For restarting VS Code, that's here https://github.com/microsoft/vscode/issues/20013

For restarting the OS and restoring a layout, this is deferred to extensions like [this](https://marketplace.visualstudio.com/items?itemName=fabiospampinato.vscode-terminals). They may do something like this now but there is only the split command, as such they might be blocked on a split extension api covered in https://github.com/microsoft/vscode/issues/45407

**Comment by bounty1342** [comment]
Was speaking about the layout.

https://marketplace.visualstudio.com/items?itemName=fabiospampinato.vscode-terminals will not bring the terminal as in the pictures.

**Comment by Tyriar** [comment]
@bounty1342 ah ok, then you're after #45407. That API is blocked on terminal tabs happening as whatever we do there will inform how the API works https://github.com/microsoft/vscode/issues/10546
